ckanapi/datapackage.py

```diff
@@ -137,6 +137,25 @@ def populate_datastore_res_fields(ckan, res):
         return  # with localckan we'll get the real CKAN exception not a CKANAPIError subclass
     res['datastore_fields'] = ds['fields']
 
+def populate_datastore_res_views(ckan, res):
```

Comment:
the `datapackage.py` module is about a specific thing called "datapackages" this function would be better in the `dump.py` module.

Reply:
@wardi populate_datastore_res_views method has been moved to the dump.py module now.